src/screens/ChannelListScreenV2/IncognitoEmptyChat/index.tsx

```diff
@@ -107,24 +106,20 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontSize: normalizeFontSize(16),
     lineHeight: normalizeFontSize(24),
-    color: COLORS.anon_primary,
+    color: COLORS.white2,
     paddingHorizontal: dimen.normalizeDimen(40),
     textAlign: 'center',
-    marginTop: dimen.normalizeDimen(16)
+    marginTop: dimen.normalizeDimen(8)
   },
   wrapper: {
+    marginTop: dimen.normalizeDimen(24),
     paddingHorizontal: dimen.normalizeDimen(20),
     width: '100%'
   },
-  line: {
-    height: 2,
-    backgroundColor: COLORS.gray200,
-    marginBottom: dimen.normalizeDimen(24)
-  },
   item: {
     flexDirection: 'row',
     alignItems: 'center',
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.anon_primary` to `COLORS.white2` and the background color change from `COLORS.white` to `COLORS.gray100` might affect the visibility and readability of the text. Please ensure that these changes adhere to accessibility standards for contrast ratios.



<!-- This is an auto-generated comment by OSS CodeRabbit -->